package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "npm run lint && babel src --out-dir lib",
     "clean": "rimraf ./lib",
-    "lint": "eslint src",
+    "lint": "node node_modules/.bin/eslint src",
```

Comment:
I think there is no difference with this change.

Reply:
I don't remember the exact reason for this, removed it from the PR